src/System.Collections.NonGeneric/tests/Performance/Perf.HashTable.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+using Microsoft.Xunit.Performance;
+using System.Collections.Generic;
+
+namespace System.Collections.Tests
+{
+    public class Perf_HashTable : PerfTestBase
+    {
+        private IEnumerable<Hashtable> TestHashtables()
+        {
+            yield return CreateHashtable(100);
+            yield return CreateHashtable(1000);
+        }
+
+        private Hashtable CreateHashtable(int size)
+        {
+            Hashtable ht = new Hashtable();
+            for (int i = 0; i < size; i++)
+                ht.Add(CreateString(50), CreateString(50));
+            return ht;
+        }
+
+        [Benchmark]
+        public void ctor()
+        {
+            foreach (var iteration in Benchmark.Iterations)
+                using (iteration.StartMeasurement())
+                    new Hashtable();
```

Comment:
Is the overhead of StartMeasurement and its IDisposable really low enough that we can measure the small cost of Hashtable's ctor appropriately and without lots of noise?  I'd have expected we'd have needed a loop inside even a single iteration in order to get something meaningful.